test/Qowaiv.UnitTests/TelephoneNumberTest.cs

```diff
@@ -0,0 +1,632 @@
+﻿using NUnit.Framework;
+using Qowaiv.Globalization;
+using Qowaiv.TestTools;
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Runtime.Serialization;
+using System.Xml.Serialization;
+
+namespace Qowaiv.UnitTests
+{
+    /// <summary>Tests the telephone number SVO.</summary>
+    public class TelephoneNumberTest
+    {
+        /// <summary>The test instance for most tests.</summary>
+        public static readonly TelephoneNumber TestStruct = TelephoneNumber.Parse("+31 (0)123.456.789");
+
+        /// <summary><see cref = "TelephoneNumber.Empty"/> should be equal to the default of telephone number.</summary>
+        [Test]
+        public void Empty_EqualsDefault()
+        {
+            Assert.AreEqual(default(TelephoneNumber), TelephoneNumber.Empty);
+        }
+
+        /// <summary>TelephoneNumber.IsEmpty() should be true for the default of telephone number.</summary>
+        [Test]
+        public void IsEmpty_Default_IsTrue()
+        {
+            Assert.IsTrue(default(TelephoneNumber).IsEmpty());
+        }
+
+        /// <summary>TelephoneNumber.IsEmpty() should be false for TelephoneNumber.Unknown.</summary>
+        [Test]
+        public void IsEmpty_Unknown_IsFalse()
+        {
+            Assert.IsFalse(TelephoneNumber.Unknown.IsEmpty());
+        }
+
+        /// <summary>TelephoneNumber.IsEmpty() should be false for the TestStruct.</summary>
+        [Test]
+        public void IsEmpty_TestStruct_IsFalse()
+        {
+            Assert.IsFalse(TestStruct.IsEmpty());
+        }
+
+        /// <summary>TelephoneNumber.IsUnknown() should be false for the default of telephone number.</summary>
+        [Test]
+        public void IsUnknown_Default_IsFalse()
+        {
+            Assert.IsFalse(default(TelephoneNumber).IsUnknown());
+        }
+
+        /// <summary>TelephoneNumber.IsUnknown() should be true for TelephoneNumber.Unknown.</summary>
+        [Test]
+        public void IsUnknown_Unknown_IsTrue()
+        {
+            Assert.IsTrue(TelephoneNumber.Unknown.IsUnknown());
+        }
+
+        /// <summary>TelephoneNumber.IsUnknown() should be false for the TestStruct.</summary>
+        [Test]
+        public void IsUnknown_TestStruct_IsFalse()
+        {
+            Assert.IsFalse(TestStruct.IsUnknown());
+        }
+
+        /// <summary>TelephoneNumber.IsEmptyOrUnknown() should be true for the default of telephone number.</summary>
+        [Test]
+        public void IsEmptyOrUnknown_Default_IsFalse()
```

Comment:
IsTrue

Reply:
Decided not to continue with this for now.